cpp/include/cugraph/graph_view.hpp

```diff
@@ -661,6 +664,21 @@ class graph_view_t<vertex_t,
   edge_t count_self_loops(raft::handle_t const& handle) const;
   edge_t count_multi_edges(raft::handle_t const& handle) const;
 
+  void attach_mask(std::vector<graph_mask_t<vertex_t, edge_t>*>& graph_mask)
```

Comment:
Should we better attach a view object?

Reply:
I do think attaching the view object here can make more sense- that way we can assume the necessary mask elements (e.g. vertex / edge mask) have been properly allocated before being used in a `graph_view` object.